deploy/gk-deploy

```diff
@@ -489,18 +489,32 @@ if [[ $? -ne 0 ]]; then
 fi
 output "OK"
 
+s=0
 heketi_service=""
 debug -n "Determining heketi service URL ... "
 while [[ "x${heketi_service}" == "x" ]] || [[ "${heketi_service}" == "<none>" ]]; do
-  heketi_service=$(${CLI} describe svc/deploy-heketi | grep "Endpoints:" | awk '{print $2}')
+  if [[ ${s} -ge ${WAIT} ]]; then
+    debug "Timed out waiting for deploy-heketi service."
+    break
+  fi
   sleep 1
+  ((s+=1))
+  if [[ "${CLI}" == *oc\ * ]]; then
+    heketi_service=$(${CLI} describe routes/deploy-heketi | grep "Requested Host:" | awk '{print $3}')
```

Comment:
It will always have the URL of the route. 

```
Name:                   heketi
Namespace:              storage
Created:                32 hours ago
Labels:                 glusterfs=heketi-route
                        template=heketi
Annotations:            openshift.io/host.generated=true
Requested Host:         heketi-storage.cloudapps.mystorage.com
                          exposed on router storage-router 32 hours ago
                          exposed on router router 32 hours ago
Path:                   <none>
TLS Termination:        <none>
Insecure Policy:        <none>
Endpoint Port:          <all endpoint ports>

Service:        heketi
Weight:         100 (100%)
Endpoints:      10.130.0.17:8080
```

This is someone else's setup and I cannot redo gk-deploy now. I am sure that the requested host line always stays this way even when Endpoints: is either \<none\> or "".